specification/paths/Shops-shop_id-integrations-integration_id-shipments.json

```diff
@@ -0,0 +1,160 @@
+{
+  "parameters": [
+    {
+      "$ref": "#/components/parameters/path-shop_id"
+    },
+    {
+      "$ref": "#/components/parameters/path-integration_id"
+    }
+  ],
+  "get": {
+    "tags": [
+      "Shops"
+    ],
+    "parameters": [
+      {
+        "$ref": "#/components/parameters/query-filter-created_at-date_from"
+      },
+      {
+        "$ref": "#/components/parameters/query-filter-created_at-date_to"
+      }
+    ],
+    "security": [
+      {
+        "OAuth2": [
+          "shipments.manage",
+          "organizations.manage",
+          "experimental"
```

Comment:
Experimental is used for when we do not yet assign other scopes. We should drop experimental or drop the other two scopes. If you want the endpoint to have the experimental warning icon, this can be done using the `src/myparcelcom.js` file.